sunspot/lib/sunspot/schema.rb

```diff
@@ -89,7 +89,13 @@ def add_filter(filter)
     #
     def to_xml
       template = File.join(File.dirname(__FILE__), '..', '..', 'templates', 'schema.xml.erb')
-      ERB.new(File.read(template), nil, '-').result(binding)
+
+      match = ERB.version.match(/\Aerb\.rb \[(?<version>[^ ]+) /)
+      if match && match[:version] >= '2.2.0' # Ruby 2.6+
```

Comment:
@koic  can we simply check for a `RUBY_VERSION` here instead of doing regexp match?

Reply:
Yep. It can be rewritten to `RUBY_VERSION`. I'll change it.